src/main/proto/th2_grpc_check1/check1.proto

```diff
@@ -23,13 +23,13 @@ import "th2_grpc_common/common.proto";
 option java_multiple_files = true;
 option java_package = "com.exactpro.th2.check1.grpc";
 
-
 service Check1 {
     rpc createCheckpoint (CheckpointRequest) returns (CheckpointResponse) {}
     rpc submitCheckRule (CheckRuleRequest) returns (CheckRuleResponse) {}
     rpc submitCheckSequenceRule(CheckSequenceRuleRequest) returns (CheckSequenceRuleResponse) {}
     rpc submitNoMessageCheck(NoMessageCheckRequest) returns (NoMessageCheckResponse) {}
     rpc waitForResult(WaitForResultRequest) returns (WaitForResultResponse) {}
+    rpc multiSubmitRules(MultiSubmitRulesRequest) returns (MultiSubmitRulesResponse) {}
```

Comment:
Please rename to submitMultipleRules(MultiRulesRequest) returns (MultiRulesResponse)